lib/internal/Magento/Framework/composer.json

```diff
@@ -20,7 +20,8 @@
         "ext-openssl": "*",
         "lib-libxml": "*",
         "ext-xsl": "*",
-        "symfony/process": "~2.1"
+        "symfony/process": "~2.1",
+        "zendframework/zend-http": "~2.4.6"
```

Comment:
I think no need to prevent updating to 2.5 and more. So better to use `~2.4`. If it should be more than 2.4.6 - `~2.4 >=2.4.6`